src/main/java/gov/usds/case_issues/config/DemoUserLoginConfig.java

```diff
@@ -58,6 +62,12 @@ public UserDetailsService getUserService() {
 					.build())
 				.collect(Collectors.toList()
 		);
+
+		for (UserDetails u : users) {
+			OAuthUser user = new OAuthUser(u.getUsername(), u.getUsername());
```

Comment:
We should give our fake users fake names, to make the tests more informative and entertaining.